openhtf/core/station_api.py

```diff
@@ -86,7 +86,7 @@
 
 # Fix for xmlrpclib to use <i8> for longs and ints instead of <int>, because our
 # timestamps are in millis, which are too big for 4-byte ints.
-xmlrpclib.Marshaller.dispatch[long] = xmlrpclib.Marshaller.dispatch[int] = (
+xmlrpc.client.Marshaller.dispatch[int] = xmlrpc.client.Marshaller.dispatch[int] = (
```

Comment:
The same expression is used twice here, and I'm not sure what the backwards compatibility story here is since int and long are separate types on Python2. We have builtins for compatibility and the `int(123)`.

I wonder if the right thing to do here instead is:

`xmlrpc.client.Marshaller.dispatch[builtins.int] = xmlrpc.client.Marshaller.dispatch[int] = (`

That's redundant on Python3 since `builtins.int` is the same as `int` but maybe we don't care?

Reply:
Alternately maybe
```
from past.builtins import long
...
xmlrpc.client.Marshaller.dispatch[long] = ...[int] = (
```